bp_common/software/py/ucode2boot.py

```diff
@@ -0,0 +1,21 @@
+import argparse
```

Comment:
what's this file doing?

Reply:
Currently, the CCE ucode is 34 bits. Adding the .bin version of the ucode file gets incorrectly interpreted by the compiler. So, this script expands the ucode instructions to 64 bits and then converts them to hex.

Reply:
Ah gotcha, maybe add a comment for that